x-pack/test/functional_endpoint/apps/endpoint/host_list.ts

```diff
@@ -121,6 +121,9 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
       after(async () => {
         // reload the data so the other tests continue to pass
         await esArchiver.load('endpoint/metadata/api_feature');
+        await pageObjects.common.navigateToUrlWithBrowserHistory('endpoint', '/hosts');
```

Comment:
These three entries in the `after()` callback seems like mistake.

Reply:
In this test we are deleting out the data for the test so it's `after()` reloads the data.  I figured it should match the original `before()`.

Reply:
Ah. Ok. Then I'm thinking we need to restructure the `describe()`'s. Top level `before()` should not load data, and the `describe()` that expect it should do it instead.
It feels fragile for a `after()` block to setup things for the next `describe()`.

Reply:
Makes sense.  I'll change it.

Reply:
I reorganized the test script and put the main tests into their own describe block.  This removes all tests from the outer describe block and moves them all into sub-describe blocks.  This way each of the interior blocks can load/unload data as needed for their set of tests.